src/main/java/org/broadinstitute/hellbender/utils/hmm/CopyNumberTriState.java

```diff
@@ -47,9 +48,12 @@
 
     public final Allele allele;
 
-    CopyNumberTriState(final String callString, final Allele allele) {
+    public final double copyRatio;
+
+    CopyNumberTriState(final String callString, final Allele allele, final double copyRatio) {
```

Comment:
not in your commit, but i wonder why the `toCallString` getter is necessary if all fields are `public final`?